tests/test.sh

```diff
@@ -758,6 +774,54 @@ debug_test()
     start_iceccd localice --no-remote -m 2
 }
 
+zero_local_jobs_test()
+{
+    echo Running zero_local_jobs test.
+
+    reset_logs local "Running zero_local_jobs test"
+    reset_logs remote  "Running zero_local_jobs test"
+
+    kill_daemon localice
+
+    start_iceccd localice --no-remote -m 0
+
+    wait_for_proc_sleep 10 $localice_pid
+
+    libdir="${testdir}/libs"
+    rm -rf  "${libdir}"
+    mkdir "${libdir}"
+
+    reset_logs remote $GXX -Wall -Werror -c testfunc.cpp -o "${testdir}/testfunc.o"
+    echo Running: $GXX -Wall -Werror -c testfunc.cpp -o "${testdir}/testfunc.o"
+    ICECC_TEST_SOCKET="$testdir"/socket-localice ICECC_TEST_REMOTEBUILD=1 ICECC_PREFERRED_HOST=remoteice1 ICECC_DEBUG=debug ICECC_LOGFILE="$testdir"/icecc.log $valgrind "${icecc}" $GXX -Wall -Werror -c testfunc.cpp -o "${testdir}/testfunc.o"
```

Comment:
if you can detect the asan error here and add this to skipped tests I think we are good to go

skipped_tests="$skipped_tests zero_local_jobs_test"